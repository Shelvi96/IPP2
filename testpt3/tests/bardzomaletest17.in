NEW ID

4149:5558:>5511:
26211:898>4
0;>54931
1>0;6:
;0817>30
;29;60>6042
;744>887606;18
4>:92061
:493>;4;68
690>7086540973
97426>63:6225039
018;286;>:97
789423;0>16
@46255675
@1
@006:16096159881790184
@;94541
@533622786827
@37153:54793::;3707
@8:2:4
@:2:0144959086477;
@52046;922;
@:16147283;5435:;49:1206
@7581;;:21435395612:40044966470
@693;
@2137664939844:
@4;
@80;24:6;84285998476991:5:27;:6
@7;7176872412924
@;;7008:965593352214:;4
