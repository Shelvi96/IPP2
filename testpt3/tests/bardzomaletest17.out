0
0
48367692
0
0
66357
0
16117
0
72747777192
2606684384770523136
0
10
0
556405692410060339
72
6062314766
